### DISCUSSION TITLE
Overview rewards GG

### DISCUSSION THREAD

**Discussion by Drysal** [mainIine/foe-helfer-extension#3129] [Ideas]
After the guild battle update, the rewards are only displayed as badges. But I would still like to know what I have received. It would be nice to see an overview in the foe helper similar to the daily player overview

**Comment by outoftheline** [comment]
lucky you, there is already the stats module:
![image](https://github.com/user-attachments/assets/9a9ac99e-f645-4378-bca5-369a96990d85)
![image](https://github.com/user-attachments/assets/4a66e5f9-85b1-43f1-b3aa-8767b79c930a)
